### ISSUE TITLE
History graph on entity is blurred

### ISSUE THREAD

**Issue by bigboban** [home-assistant/frontend#14295]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

Graph on Info tab under value entity is "blurred" or "foggy" on new HA 2022.11.x. Yeas i understand this:

_The “more info” dialogs of entities now use long-term statistics for the graphs shown. It makes those dialogs a lot faster, especially on mobile devices._

![image](https://user-images.githubusercontent.com/7159224/200045288-c2a6ea8a-2f1d-48fe-a3f1-75717e706d84.png)

But i hate this "improvement" if it is default behavior and cannot be changed. All graphs are now awful. Yes i can strikethrough min and max and let only mean. But this option is NOT saved and after reopening graph is awful again.

PLEASE fix this! Let us to choose default view (hide min/max for example). 

### Describe the behavior you expected

Graph is clear as before.

### Steps to reproduce the issue

1. Click on some temp sensor.
2.  View INFO tab (some line graph)
3.
...

### What version of Home Assistant Core has the issue?

2022.11.1

### What was the last working version of Home Assistant Core?

2022.10.x

### In which browser are you experiencing the issue with?

Google chrome

### Which operating system are you using to run this browser?

Windows 11

**Comment by ildar170975** [comment]
> Yes i can strikethrough min and max and let only mean

The "mean" value is not a precise REAL value - it is an average one.
So, the stat graph is absolutely not the same as the old history-graph.
This may be accepted only if you are not interested in a real data.

**Comment by bigboban** [comment]
Yes i understand. But for measures like temperature (small/slow changes) is mean almost same as actual value. BUT. This is main design problem not precision problem. New design is very very ugly in default setting (min/max/mean all on).

Please fix this somehow.

Some strange point - why some entities has this new view (statistics) and some has old view?

**Comment by ildar170975** [comment]
> New design is very very ugly in default setting (min/max/mean all on).

https://github.com/home-assistant/frontend/pull/14273#event-7746118178



> why some entities has this new view (statistics) and some has old view?

Probably because these sensors do not have LTS?

**Comment by bigboban** [comment]
https://github.com/home-assistant/frontend/pull/14273 means graph will be still blurry but without option to hide min/max?

**Comment by ildar170975** [comment]
> graph will be still blurry

Unclear what yo mean by "blurry".
IMHO https://github.com/home-assistant/frontend/pull/14273 means that min & max graph will be shown w/o lines but with colored areas between lines.

**Comment by bigboban** [comment]
![image](https://user-images.githubusercontent.com/7159224/200141216-132dbbe8-d141-453e-af12-43debc930e78.png)

**Comment by ildar170975** [comment]
It is not blurred - it is just several graphs displayed near each other.

**Comment by bigboban** [comment]
No matter why but IT IS blurred. I understand technical background of course. But important is result. And result is for 90% users worse than previous "clear" graph. Again - let is as OPTION but don't force this for everyone please and don't let is as default.

**Comment by bigboban** [comment]
Next proof new look is nonsense. My Xiaomi air purifier has "time use" sensor. It's value only go up one step (by one). It takes no sense to "blur" line, show some min/max is totally unusable. I have maybe one or two sensors from total tens of sensors where min/max makes some sense. 

![image](https://user-images.githubusercontent.com/7159224/200180712-539b1463-d261-4add-b3fa-dbd42d8c443d.png)

**Comment by Blacksli** [comment]
Please bring back HISTORY!!! Not statistics, but HISTORY!!!

**Comment by LordBoos** [comment]
I'd like to have a toggle (which state is saved between HA restarts) that switches between the old history and new statistics per entity. I agree with @bigboban, from tens of sensors that I use this new statistics view makes sense for like 3 or 4 of them.

**Comment by CarlosGS** [comment]
The graphs are not only blurred, but also less accurate as decimal places are discarded, see https://github.com/home-assistant/frontend/issues/14218#issuecomment-1337111886.
Please vote on that thread:

- https://github.com/home-assistant/frontend/issues/14218

**Comment by bigboban** [comment]
Still broken. When will be old and **correct** view back?
